src/test-utils/dom/prompt-input/index.ts

```diff
@@ -13,16 +13,27 @@ export default class PromptInputWrapper extends ComponentWrapper {
     return this.findByClassName<HTMLTextAreaElement>(testutilStyles.textarea)!;
   }
 
+  /**
+   * Finds the primary action button. Note that if you provide `primaryAction`
+   * then this will return null, you should use `findPrimaryActionSlot` instead.
+   */
   findActionButton(): ElementWrapper<HTMLButtonElement> {
     return this.findByClassName<HTMLButtonElement>(testutilStyles['action-button'])!;
   }
 
+  /**
```

Comment:
unfortunately we got a lot of these typings wrong on the first release :( and teams are already using them so we can't easily add `| null`